pod/perldelta.pod

```diff
@@ -356,7 +356,9 @@ well.
 
 =item *
 
-XXX
+The XS parser, L<ExtUtils::ParseXS> has been extensively restructured
```

Comment:
This looks weird to me: There is a comma before `ExtUtils::ParseXS`, but not after. Shouldn't it be symmetrical? (Asking people who understand English.)